specification/compute/resource-manager/readme.az.md

```diff
@@ -2,14 +2,60 @@
 
 These settings apply only when `--az` is specified on the command line.
 
-``` yaml $(az) && $(target-mode) == "core"
+``` yaml $(az) && $(cloudservice)
+tag: package-2020-10-01-preview-only
+az:
+  extensions: cloud-service
+  namespace: azure.mgmt.compute
+  package-name: azure-mgmt-compute
+az-output-folder: $(azure-cli-extension-folder)/src/cloudservice
+python-sdk-output-folder: $(az-output-folder)/azext_cloudservice/vendored_sdks/cloudservice
+cli:
+    cli-directive:
+        - where:
+            group: "*"
+            op: "*"
+          hidden: true
+        - where:
+            group: "CloudServiceRoleInstances"
+            op: "*"
+          hidden: false
+        - where:
+            group: "CloudServiceRoles"
+            op: "*"
+          hidden: false
+        - where:
+            group: "CloudServices"
+            op: "*"
+          hidden: false
+        - where:
+            group: "CloudServicesUpdateDomain"
+            op: "*"
+          hidden: false
+directive:
+  - where:
+      group: cloud-service cloud-service-role-instance
+    set:
+      group: cloud-service role-instance
+  - where:
+      group: cloud-service cloud-service-role
+    set:
+      group: cloud-service role
+  - where:
+      group: cloud-service cloud-service-update-domain
+    set:
+      group: cloud-service update-domain
+```
+
+``` yaml $(az) && $(target-mode) == "core" && $(sshkey)
+tag: package-2020-12-01-only
 az:
   extensions: vm
   namespace: azure.mgmt.compute
   package-name: azure-mgmt-compute
 az-output-folder: $(azure-cli-folder)/src/azure-cli/azure/cli/command_modules/vm
 python-sdk-output-folder: "$(az-output-folder)/vendored_sdks/vm"
-compatible-level: track2
```

Comment:
compatible-level: track2 should be necessary, do you find any issue for it?

Reply:
I specify it in command line
```
--compatible-level=track2
```

Reply:
depth coverage pipeline need this,

Reply:
added now